plugins/turing/runner/experiment_runner_test.go

```diff
@@ -180,6 +180,10 @@ func TestFetchTreatment(t *testing.T) {
 									},
 									"name": "test_experiment-control",
 									"traffic": 50
+								},
+								"metadata": {
+									"experiment_version": 1,
+									"experiment_type": "a/b"
```

Comment:
I think the expected value should be `"A/B"` right? https://github.com/caraml-dev/xp/blob/main/api/schema.yaml#L368

Reply:
It's only a mocked test, but I updated it to match the treatment service response anyway. Thanks!